packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md

```diff
@@ -128,37 +130,55 @@ You can modify the Application's Sign-On Policy to require the user to have a se
+7. Click <StackSnippet snippet="tryenrollout" inline /> to sign out of the <StackSnippet snippet="applang" inline /> app.
+
+## Self-service Password Recovery
 
-## Authenticator recovery
+> **Note:** Self-service Password Recovery uses the **Forgot Password** email template. You can edit this template. See [Edit a default email template](/docs/guides/custom-email/main/#edit-a-default-email-template).
 
-In your org, Password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
+### Email magic link password recovery flow
+
+> **Note:** In your org, password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
+
+Try the email magic link recovery flow:
+
+1. Select **Forgot password?** in the Sign-In Widget.
+1. Enter your email or username when prompted and click **Next**.
+1. Click **Send me an email**. A magic link is sent to your email address.
+1. Click **Reset password**. A new window opens. Click **Yes, it's me**.
+1. Enter a new password. Click **Reset Password**.
+
+### One-time password recovery flow
+
+> **Note:** In your org, password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
 
 Try out the email password recovery flow:
 
 1. Select **Forgot password?** in the Sign-In Widget.
 1. Enter your email or username when prompted and click **Next**.
-1. Click **Select** for the Email authenticator. An OTP code is sent to your email address.
-1. Manually copy the code from the email and paste it into the Sign-In Widget.
-1. After you paste the code, answer the security question that appears. You are then prompted to enter a new password.
-1. After you enter the new password successfully, you are prompted for the additional phone authentication that you set up in the last section. Then, you are redirected to the <StackSnippet snippet="applang" inline /> welcome page.
+1. Click **Select** for the Email authenticator. An OTP code is sent to your email address. Manually copy the code from the email.
+1. In the widget, click **Enter a code from the email instead**, paste the code and click **Verify**.
+1. Enter a new password.
+1. After you enter the new password successfully, you are prompted for the additional phone authentication that you set up in [Enable multifactor authentication](#enable-multifactor-authentication). Then, you are redirected to the <StackSnippet snippet="applang" inline /> welcome page.
 1. Click <StackSnippet snippet="tryenrollout" inline /> to sign out of the <StackSnippet snippet="applang" inline /> app.
 
-### Recovery with Okta Verify
+### Okta Verify recovery flow
 
 In addition to recovering your password with an email, you can add Okta Verify as a recovery option.
 
+<!-- Needs a link to verify setup as recovery option -->
```

Comment:
Does this comment refer to a change that needs to happen to the content post publish?

Reply:
Yes.